libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -36,18 +57,23 @@ object MapboxMetricsReporter : MetricsReporter {
         accessToken: String,
         userAgent: String
     ) {
-        mapboxTelemetry = MapboxTelemetry(context, accessToken, userAgent)
-        mapboxTelemetry.enable()
+        eventsService = EventsServiceProvider.provideEventsService(
+            EventsServerOptions(accessToken, userAgent, null)
+        )
+        eventsService.registerObserver(eventsServiceObserver)
+
+        if (!TelemetryUtilsDelegate.getEventsCollectionState()) {
```

Comment:
we can remove

```
        if (!TelemetryUtilsDelegate.getEventsCollectionState()) {
            TelemetryUtilsDelegate.setEventsCollectionState(true)
        }
```

see [thread1](https://github.com/mapbox/mapbox-maps-android/pull/1672#discussion_r974249473)
and [thread2](https://github.com/mapbox/mapbox-maps-android/pull/1672#discussion_r976230286)
for reference

Reply:
or in case it's planned to enable telemetry on `init`(because there is no `enable` method) we should not check `getEventsCollectionState`

`getEventsCollectionState`/`setEventsCollectionState` reads/writes persistent settings

Reply:
discussed internally: this is common setting for all SDKs and should be controlled out of `MapboxMetricsReporter`, remove this from MapboxRerpoter class